### ISSUE TITLE
Disable updating a feed?

### ISSUE THREAD

**Issue by KernelLeak** [brief-rss/brief#544]
I recently noticed that a feed that I had subscribed to fell into a spammer's hands, who promptly used it to send spam - so I looked for an option to disable updating that feed but to keep the old, existing entries - but I can't say that I could find one, so I reluctantly nuked the feed...

Is there currently a way to prevent Brief from updating a single feed? If not, an "Enabled" option for feeds might be a useful feature...

**Comment by KernelLeak** [comment]
Yeah, that should do it, thanks - but... a real toggle, with an indication in the list (dimmed feed name?) would still be great... :)

**Comment by tanriol** [comment]
Duplicate of #349.
